serving-runtimes/vllm_runtime/README.md

```diff
@@ -2,7 +2,8 @@
 
 The [vLLM](https://docs.vllm.ai/en/latest/index.html) runtime can be used with Open Data Hub and OpenShift AI Single-Model Serving stack to serve Large Language Models (LLMs) as an alternative to Caikit+TGIS or standalone TGIS. Currently supported models are listed [here](https://docs.vllm.ai/en/latest/models/supported_models.html).
 
-Note that as this runtime is specifically meant to run LLMs and uses custom kernels based on CUDA, a **GPU is required** to load models.
+Note that as the standard runtime is specifically meant to run LLMs and uses custom kernels based on CUDA, a **GPU is required** to load models for this version.  
+There is. however, also a **CPU version** that can load models on the CPU.
```

Comment:
typo: There is, however, also a **CPU version** that can load models on the CPU.

Reply:
Good catch, thanks